src/pages/reservation-history/ReservationHistory.tsx

```
@@ -18,6 +18,7 @@ import {
 import ReservationList from '@pages/reservation-history/components/ReservationList';
 
 export default function ReservationHistory() {
+  const navigate = useNavigate();
```

ESLint

[error] 'navigate' is assigned a value but never used.

(no-unused-vars)